src/test/java/org/opentripplanner/apis/gtfs/GraphQLIntegrationTest.java

```diff
@@ -216,13 +218,21 @@ public TransitAlertService getTransitAlertService() {
     var alerts = ListUtils.combine(List.of(alert), getTransitAlert(entitySelector));
     transitService.getTransitAlertService().setAlerts(alerts);
 
+    DefaultVehicleRentalService defaultVehicleRentalService = new DefaultVehicleRentalService();
+    VehicleRentalStation vehicleRentalStation = new TestVehicleRentalStationBuilder()
+      .withLatitude(1)
+      .withLongitude(1)
```

Comment:
Can you do me a favour? Can you replace this with a newly added method `withCoordinate(1,1)`? If you want to be extra nice, you can change all the call sites and remove the `withLat/Long` methods.

Reply:
I made this change, though actually ended up removing the explicit setting of coordinates for this test case, since the default coordinates work just as well. I did also remove these methods and changed other usages to `withCoordinates`.